New package: DataMigrations v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: DataMigrations
- Repository: https://github.com/AlgebraicJulia/DataMigrations.jl
- Created by: @KevinArlin
- Version: v0.0.1
- Commit: 7d12e04966a5d7bb61a854eb86294ea1c42f52dc
- Reviewed by: @KevinArlin
- Reference: https://github.com/AlgebraicJulia/DataMigrations.jl/issues/150#issuecomment-1874388365
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a8359884bdeeae60f2a9dad397bf90d999d5115f0543771391e66858a924610c753b7db474cddbb3462252515f0f18982e38874b7b599e94fa85c0f562837379995776a5fd2c43dc095b488919d4eb7ecd0f938a89fcb52dd9d90f204b23d0dc6f7a4ab8e82864c7b1710b7e78500155c02fa09dfa4c3e0d84d72811fc9bec497aae9dc102ea223fcebee24be7ffde91eb7906 -->

KevinArlin commented:
I'd like to register this package but it's dependent on beta Julia. It looks like that's inconsistent with AutoMerge.

goerz commented:
I’d say a package that depends on an unreleased version of Julia shouldn’t be in the General registry. Use a LocalRegistry for now and try registering again when Julia 1.10 is out

JuliaRegistrator committed:
New package: DataMigrations v0.0.1

UUID: 0c4ad18d-0c49-4bc2-90d5-5bca8f00d6ae
Repo: https://github.com/AlgebraicJulia/DataMigrations.jl.git
Tree: 833489863fe94d1137eb9d3dcbceb45ca6d7d3d0

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
